src/components/Card/Card.scss

```
@@ -1,47 +1,92 @@
 @import '../../styles/variable.scss';
-
-.cardContainer {
+@mixin btn {
+  padding: 10px 0;
+  border: 1px solid black;
   cursor: pointer;
-
+}
+.cardContainer {
+  margin: 0 auto;
   .cardImage {
+    position: relative;
     width: 300px;
     height: 300px;
     cursor: pointer;
-
-    img {
+    .thumbnailImage {
       width: 100%;
       height: 100%;
       object-fit: cover;
     }
+    .hoverImage {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      opacity: 0;
+      transition: opacity 0.5s linear;
+      object-fit: cover;
+    }
+    &:hover .hoverImage {
+      opacity: 1;
+    }
   }
-
   .cardTag {
     @include display-flex(space-between, center);
-    width: 300px;
     padding-top: 10px;
-
+    width: 300px;
     .cheeseName {
       flex: 7;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
     }
-
     .cheesePrice {
       flex: 3;
       text-align: right;
       font-weight: 500;
     }
   }
-
   .starRatings {
     padding: 10px 0;
+    i {
+      color: #fec600;
+      padding-right: 5px;
+    }
   }
-
-  button {
-    width: 300px;
-    padding: 18px 0;
-    border: 1px solid black;
-    cursor: pointer;
+  .btnWrapper {
+    opacity: 0;
+    .hide {
+      display: none;
+    }
+    .btnStatic {
+      @include btn;
+      width: 300px;
+      transition: opacity 0.5s linear;
+    }
+    .btnOptionWrapper {
+      @include display-flex(space-between, center);
+      margin-bottom: 5px;
+      .btnOption {
```

```suggestion
    .btnOptionWrapper {
      @include display-flex(space-between, center);
      margin-bottom: 5px;
      
      .btnOption {
```
새로운 셀렉터 앞에서는 한칸 꼭... 띄워주세요...

---

아앗... 주의하겠습니다.